sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncUserDefinedFunction.java

```diff
@@ -115,4 +121,45 @@ String getLink() {
         builder.append(getId());
         return builder.toString();
     }
+
+    private Mono<CosmosAsyncUserDefinedFunctionResponse> readInternal(Context context) {
+        String spanName = "readUDF." + container.getId();
```

Comment:
The span name should match the method name which is `read` here

Reply:
Same for methods `replace` and `delete` below.

Reply:
Discussed offline , we need resource name on read/delete/replace as we have hierarchy model and all the resource have same api name for these operation, so we need to differentiate them